### PR TITLE
Added Breast Cancer Detection Script

### PR DESCRIPTION
**Pull request by Debajyoti111**
`Please fill all the details and checkbox, so we can merge PR faster.`

# Pull Request Template

**script name** -
breast_cancer_detection/detection.py
**What have you Changed**
I have added the breast_cancer_detection_folder in which there is a detection.py script, a README.md file and a requirements.txt file

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder
- [x] I am following clean code and Documentation and my code is well-linted with flake8.
- [x] I have added README.md and requirements.txt with my script

If the issue was not assigned to you Please don't make a PR. It will mark as invalid.

Join Us on Discord:- https://discord.gg/JfbK3bS

## acknowledgment
- [x] I have tested the script and distributed it under an MIT license.

### CODE REVIEW

**Comment by Debajyoti111** [comment]
Hi, I am working on ISSUE #543, till then please review it.

File: breast_cancer_detection/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+1. Import the following modules to run the script-
```
Comment (codeperfectplus) [inline breast_cancer_detection/requirements.txt]: your requirements.txt is not in correct format

**Review by codeperfectplus — CHANGES_REQUESTED** [review]

File: breast_cancer_detection/requirements.txt (hunk 1 above)
Comment (Debajyoti111) [reply breast_cancer_detection/requirements.txt]: Alright fixing it!

**Comment by Debajyoti111** [comment]
@codePerfectPlus I have updated the code! Please review!

**Review by codeperfectplus — APPROVED** [review]
Thanks for making your contributions.

you can join us on telegram for more updates about Hacktoberfest.
Hacktoberfest Channel:- t.me/hacktober2022

Happy hacktoberfest

**Pull request merged**